chia-blockchain/templates/monit_notify.sh.j2

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Webhook URL where the alert will be sent
+WEBHOOK_URL="https://alert-receiver.chiaops.com/ips-info"
```

Comment:
Since these are open source roles, this should be a var

Reply:
Switched to using a var